cmds/Makefile

```diff
@@ -8,10 +8,13 @@ ldflags = '-w -s -X $(version).Branch=$(branch) -X $(version).Revision=$(revisio
 all: $(shell ls -d */)
 	strip $(OUT)/*
 
-.PHONY: output clean
+.PHONY: output clean identityd
 
 output:
 	mkdir -p $(OUT)
 
+identityd:
+	cd identityd && GOOS=linux go build -ldflags $(ldflags) -o $(OUT)/identityd
```

Comment:
this is needed cause you now need gco ?
isn't cgo only used by the fuse part of 0-Fs ? I guess you do not need it just to read the flist is it ?

Reply:
No actually this is needed by sqlite !!